cds/modules/theme/static/js/cds/module.js

```diff
@@ -225,6 +225,13 @@ app.filter('isPublic', function () {
   };
 });
 
+app.filter('defaultYear', function () {
+  return function (year) {
```

Comment:
year.getFullYear() would work? it is a string, no?

Reply:
You mean `return year.getFullYear();` instead of `return (year || (new Date()).getFullYear());` ? The idea of this filter is to return the current year if the `year` is undefined or empty or something.